samples/hybrid-connections/java/simple-websocket-demo/README.md

```diff
@@ -0,0 +1,90 @@
+# 'Simple' Web Socket Hybrid Connection Sample 
+
+First, as described in the description [Here](https://github.com/bainian12345/azure-relay/tree/java-samples/samples/hybrid-connections/java), you will need to retrieve different parameters from the enviroment variable `RELAY_CONNECTION_STRING` for connection and authentication purposes:
```

Comment:
>you will need to  [](start = 142, length = 17)

I kind of think "you will need to" sounds like they must take additional steps, I might simply say "the sample code retrieves the connection parameters"